src/main/java/org/folio/circulation/domain/policy/Policy.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.circulation.domain.policy;
+
+public abstract class Policy {
```

Comment:
i'd suggest to rename this class to CirculationPolicy to improve readability and maintainability.
Policy class already exists in circulation

Reply:
Renamed